src/utils/nameModifiers.ts

```diff
@@ -7,6 +7,9 @@
 // Used in TupleFiller in TupleFixes
 export const TUPLE_FILLER_PREFIX = '__warp_tf';
 
+// Used in librariesConverter
+export const LIBRARY_CONTRACT_INFIX = 'library_';
```

Comment:
This is actually used as a prefix, should consider changing the name to `LIBRARY_CONTRACT_PREFIX`